config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -153,7 +153,7 @@ susy.quest.db.47.desc=The §7LV Bending Machine §ris a better method for making
 susy.quest.db.48.title=Rubber Plates
 susy.quest.db.48.desc=Use the Vulcanizing Press to create §6Rubber Sheets.
 susy.quest.db.49.title=Coagulation Tank
-susy.quest.db.49.desc=The§6 Coagulation Tank§r can coagulate fluids into solids.%n%nAt minimum, the Coagulation Tank requires:%n§6- 1 Coagulation Tank Controller%n- 24 Wooden Coagulation Tank Wall%n§r%nThe tank can coagulate liquid Latex into a solid hunk of Latex. %nTo make the process faster, you can either use some §6Catalysts§r to boost the processing speed, or extend its structure to make it parallel. %n%nYou can §cautomate this machine §rby adding a GT Input Hatch and Output Bus to the structure.%n%nWhile this machine isn't strictly required, it makes the next step of the Rubber processing chain about §c8x more efficient§%n%n%n§f§lNote: This multiblock has some issues with input hatch/output bus placement. I suggest you try building this multiblock in creative first, to see which hatch placment allows the §6§lCoagulation Tank§r§l to form.
+susy.quest.db.49.desc=The§6 Coagulation Tank§r can coagulate fluids into solids.%n%nAt minimum, the Coagulation Tank requires:%n§6- 1 Coagulation Tank Controller%n- 24 Wooden Coagulation Tank Wall%n§r%nThe tank can coagulate liquid Latex into a solid hunk of Latex. %nTo make the process faster, you can either use some §6Catalysts§r to boost the processing speed, or extend its structure to make it parallel. %n%nYou can §cautomate this machine §rby adding a GT Input Hatch and Output Bus to the structure.%n%n§lThis machine is necessary to make latex.§r
```

Comment:
maybe it's not necessary to say it's needed for the latex?
Just not saying it's optional is enough ig